insights/specs/__init__.py

```diff
@@ -621,6 +621,7 @@ class Specs(SpecSet):
     rhsm_katello_default_ca_cert = RegistryPoint()
     rhsm_log = RegistryPoint(filterable=True)
     rhsm_releasever = RegistryPoint(no_obfuscate=['hostname', 'ip'])
+    rhui_set_release = RegistryPoint()
```

Comment:
Per the spec application MR in the assets repo, this spec should be marked as `no_obfuscate=['hostname', 'ip']`